Docker/start-container.sh

```diff
@@ -3,14 +3,14 @@
 #Set user and group
 umask 0002
 PUID=${PUID:-`id -u squeezeboxserver`}
-PGID=${PGID:-`id -g squeezeboxserver`}
+PGID=${PGID:-`id -g users`}
 
 usermod -o -u "$PUID" squeezeboxserver
-groupmod -o -g "$PGID" squeezeboxserver
+groupmod -o -g "$PGID" users
 
 umask 0002
 
 #Add permissions
-chown -R squeezeboxserver:users /config /playlist
+chown -R squeezeboxserver:users /config /playlist /music /lms
```

Comment:
`chown -R /music` will try to change all my music collection - which obviously won't work, as I mount it read only. And I certainly don't want it to change permissions anyway.

Reply:
I've removed /music as this should be RO.